src/enamel/SCALP.java

```diff
@@ -539,4 +563,27 @@ private void removeHWButtons() {
      void speak(String message) {
        voice.speak(message);
     }
+     
+     private void fileHanderInit() {

```

Comment:
The file handler function is good, but maybe we should move this to a separate class. I noticed other parts of the code that would make good use of this function.

Reply:
Maybe changing the function to receive a `this.logger` instance and a `Path` variable would make it more flexible and re-usable.

Reply:
If there are no objections, I am going to discard this PR. This method is being moved to the logger class, and the two other relevant lines in the code will be included in the other logging PR after I commit the logging class, and then adjust the logging instances across the TBB.

Reply:
Alternatively, I can update the other logging branch with the new Logger Class, merge it with the development branch, and then update this branch with its new intended code. I am happy with whichever option makes most sense from a workflow perspective.